components/atoms/ActionButton.js

```diff
@@ -0,0 +1,121 @@
+import PropTypes from "prop-types";
+
+/**
+ * Button component
+ */
+export function ActionButton(props) {
+  return (
+    <button
+      className={`flex justify-center content-center h-auto p-1 ${
+        props.fullyRounded
+          ? "rounded-full py-2 px-4"
+          : props.rounded
+          ? "rounded-md py-2 px-4"
+          : props.notRound
+          ? "rounded-sm"
+          : "rounded-md"
+      } focus:ring-1 focus:ring-black ${
+        props.invert && !props.disabled
+          ? "bg-blue-600 text-white border border-blue-600 hover:bg-blue-500"
+          : props.className
+      } ${
+        props.disabled
+          ? "bg-gray-light text-gray-600 border border-gray-md "
+          : props.className
+      } text-sm px-4 focus:outline-none`}
+      onClick={props.onClick}
+      onMouseEnter={props.onHover}
+      onMouseLeave={props.onMouseLeave}
+      id={props.id}
+      data-cy={props.id}
```

Comment:
Would like to make data-cy a separate prop here. Minor thing but its cleaner if they are seperate